frontend/components/equipment-card.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <v-card border="md" @click="console.log()">
+    <v-row class="ma-0">
+      <v-col cols="7" class="pa-0 border-e-md">
+        <v-card-title class="text-h6 text-center">
+          {{ equipment.name }}
+        </v-card-title>
+        <v-card-text class="d-flex flex-row">
+          <v-progress-circular :model-value="equipment.oee" size="150" width="20" :color="getColorState(equipment.oee, equipment.isPaused)" class="mx-auto text-h6">
+            {{equipment.oee}}%
+          </v-progress-circular>
+        </v-card-text>
+      </v-col>
+
+
+
+      <v-col cols="5" class="pa-0">
+        <v-card-text class="fill-height justify-space-between d-flex flex-column px-0">
+          <v-card
+            class="border-b-md pa-0" :style="{background: progressGradient(getColorState(equipment.quality, equipment.isPaused), equipment.performance)}">
```

Comment:
We don't get good contrast between the foreground color and bar label/percentage. Could we turn these into radial progresses and see how it looks?